apps/web/drizzle.config.ts

```diff
@@ -0,0 +1,12 @@
+import { type Config } from "drizzle-kit";
+
+import { env } from "~/env.mjs";
+
+export default {
+  schema: "./src/server/db/schema.ts",
+  driver: "mysql2",
+  dbCredentials: {
+    connectionString: env.DATABASE_URL,
+  },
+  tablesFilter: ["web_*"],
+} satisfies Config;
```

Comment:
<i class='graphite__hidden'>[Re: lines 1 to 13]<br/></i>

move to web<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/AbhiShake1/titeet/1?utm_source=unchanged-line-comment">Graphite</a>.</span>